pyaedt/modeler/schematic.py

```diff
@@ -251,7 +265,7 @@ def model_units(self, units):
         self.oeditor.SetActivelUnits(["NAME:Units Parameter", "Units:=", units, "Rescale:=", False])
 
     @pyaedt_function_handler()
-    def move(self, selections, pos, units="meter"):
+    def move(self, selections, pos, units=None):
         """Move the selections by ``[x, y]``.
```

Comment:
```suggestion
        """Move the selections by the specified ``[x, y]`` coordinates.
```
Not sure if this should be coordinates or axis or offset?